.github/workflows/data/pytest.yml

```diff
@@ -1,5 +1,5 @@
 - name: dcpy
-  calls: ["dcpy/test -s --verbose --verbose --cov-config=pyproject.toml --cov=dcpy"]
+  calls: ["dcpy/test -s --verbose --verbose --cov-config=pyproject.toml --cov=dcpy --cov-report=xml"]
```

Comment:
this change creates `./coverge.xml` after the tests are run. if we expand coverage reporting to other tests (like the QA app), we can use more a specific filename or directory to upload multiple reports

Reply:
and xml isn't the only format (html locally is cool), but it seems to be the default format for uploading to Codecov